kitsune/questions/config.py

```diff
@@ -650,6 +639,44 @@
                 ),
             },
         ),
+        (
+            "mozilla-account",
+            {
+                "name": _lazy("Mozilla Account"),
+                "subtitle": _lazy("Mozilla account is the account system for Mozilla"),
+                "extra_fields": [],
+                "tags": [],
+                "product": "mozilla-account",
+                "categories": OrderedDict(
```

Comment:
Since this is going to be supported through ZD, it can be an empty `OrderedDict([])`